src/index.js

```diff
@@ -1,8 +1,4 @@
-import lib from './libs/index.js'
 import components from './components/index.js'
 import './index.css'
 
-export default {
```

Comment:
removed this because it wasn't in use, and to avoid extra re-exports as per this article that was going around some weeks ago.